src/mono/wasm/runtime/workers/dotnet-crypto-worker.ts

```diff
@@ -0,0 +1,386 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+import { setup_proxy_console } from "../logging";
+import { InitCryptoMessageData } from "../crypto-worker";
```

Comment:
could be `import type` which might be better to guard that we're not rolling up unwanted code.